k8s/mongodb/container/docker_build_and_push.bash

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 
-docker build -t bigchaindb/mongodb:3.1 .
+docker build -t bigchaindb/mongodb:3.2 .
```

Comment:
are we changing version number for each new build?

Reply:
@shahbazn Yes! That's the whole idea.

Reply:
Yes, we can try to automate in future.

Reply:
@krish7919 we should come up with better versioning strategy. with increased development and bigger team sizes its gonna get messy really fast if we keep bumping this every time a PR gets merged. may be add build number and update the general availability image for each named release with the release cycle. but we can have further discussion on this once we move towards automating the build process.

Reply:
Yes, that should be the approach, but we have to adjust to this for now.